docker-compose.test.yml

```diff
@@ -45,6 +45,7 @@ services:
       - SSAS_PUBLIC_URL=http://ssas:3003
       - PATIENT_IDENTIFIER_MODE=MBI_MODE
       - BCDA_ENABLE_NEW_GROUP=true
+      - PRIORITY_ACO_IDS=A9990,A9991,A9992,A9993
```

Comment:
Should we also place this in `docker-compose.yml` so the system runs locally with the list in place?  These env vars are just set in the context of the test container.